src/main/java/uk/gov/pay/adminusers/resources/InviteResource.java

```diff
@@ -71,7 +72,7 @@ public InviteResource(InviteService service, InviteRequestValidator inviteValida
     }
 
     @GET
-    @Path("/{code}")
+    @Path(INVITES_RESOURCE_V1 + "/{code}")
```

Comment:
I think these days we prefer to just inline the path strings rather than using constants, as it makes it easier to read and search.

Reply:
👍